backend/apid/apid.go

```diff
@@ -194,8 +194,11 @@ func registerGraphQLService(router *mux.Router, store store.Store, url string, t
 			router.NewRoute(),
 			middlewares.SimpleLogger{},
 			middlewares.LimitRequest{},
-			// Allow requests without an access token to continue
-			middlewares.Authentication{IgnoreUnauthorized: true},
+			// TODO: Currently the web app relies on receiving a 401 to determine if
+			//       a user is not authenticated. However, in the future we should
+			//       allow requests without an access token to continue so that the
+			//       schema can be queried without authorization.
```

Comment:
You mean queried without authentication?

Reply:
Let me update that comment.